app/src/main/java/com/breadwallet/presenter/fragments/FragmentPin.java

```diff
@@ -65,7 +65,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mainLayout.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                getActivity().getFragmentManager().beginTransaction().remove(FragmentPin.this).commit();
```

Comment:
My recollection is getActivity() is deprecated?  Otherwise what is your thinking on this @andhikayuana ?

Reply:
so on android have 2 different fragment class

- [android.app.Fragment](https://developer.android.com/reference/android/app/Fragment)
- [androidx.fragment.app.Fragment](https://developer.android.com/reference/kotlin/androidx/fragment/app/package-summary)


actually `getActivity()` still there but it will return `nullable`, do you want to use it ? more [info](https://developer.android.com/reference/kotlin/androidx/fragment/app/Fragment#getActivity())

using `requireActivity()` assuming that the activity not null, if null it will be throwing an exception
https://developer.android.com/reference/kotlin/androidx/fragment/app/Fragment#requireActivity()

Reply:
Yes. This is fine for now @andhikayuana